### ISSUE TITLE
Offer a synchronous method

### ISSUE THREAD

**Issue by bugeats** [agalazis/ssm-params-convict-adapter#1]
It would be nice to be able to load the SSM values synchronously during the node process init.

**Comment by agalazis** [comment]
Hello, I apologize for the long delay in my response what would that serve? you can use await and then boot your server. Can you describe a use case where you need it to be truly synchronous?

**Comment by bugeats** [comment]
You can't actually use async/await when loading a module file without wrapping the async call in a function or some sort. This gets awkward when you're using the normal patterns for init'ing a server instance, for example.

So the purpose this would serve is to have all your config loaded synchronously before your server module continues it's bootstrapping. 

**EDIT** to be clear:
```javascript
// this doesn't work without synchronous support:
const config = await ssmAdapter.convict(schema)
app.listen(config.get('port'))
module.exports = app;
```

```javascript
// this is the work-around for async/await in a module
(function async main () {
  const config = await ssmAdapter.convict(schema)
  app.listen(config.get('port'))
});
// but of course, this is not longer a possibility:
module.exports = app; // NOPE
```

**Comment by agalazis** [comment]
yes hence the readme demonstrates how to use it bare in mind that that's just an example. I would normally have  a `bootable` server class  with an async boot function. As for the module approach I would rather just export the app in the module(without calling listen) then import and call listen in some other executable file.
